test/stale.test.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+const t = require('tap')
+const { promisify } = require('util')
+const { Cache } = require('../src/cache')
+const createStorage = require('../src/storage')
+
+const sleep = promisify(setTimeout)
+
+const { test } = t
+
+t.jobs = 3
+
+test('stale', async (t) => {
+  t.plan(6)
+
+  const cache = new Cache({
+    storage: createStorage(),
+    ttl: 10,
+    stale: 9
+  })
+
+  cache.define('fetchSomething', async (query) => {
+    t.equal(query, 42)
+    return { k: query }
+  })
+
+  t.same(await cache.fetchSomething(42), { k: 42 })
+  t.same(await cache.fetchSomething(42), { k: 42 })
+
+  await sleep(2500)
+
+  t.same(await cache.fetchSomething(42), { k: 42 })
+
+  await sleep(500)
+
+  t.same(await cache.fetchSomething(42), { k: 42 })
+})
+
+test('global stale is a positive integer', async (t) => {
+  t.plan(1)
```

Comment:
plan is not needed

Reply:
It actually is. Otherwise if no error was thrown, the test would pass.